lib/pure/strutils.nim

```diff
@@ -84,31 +84,41 @@ from std/private/strimpl import cmpIgnoreStyleImpl, cmpIgnoreCaseImpl, startsWit
 
 
 const
-  Whitespace* = {' ', '\t', '\v', '\r', '\l', '\f'}
+  Whitespace* = {' ', '\t', '\v', '\r', '\n', '\f'}
     ## All the characters that count as whitespace (space, tab, vertical tab,
     ## carriage return, new line, form feed).
 
   Letters* = {'A'..'Z', 'a'..'z'}
     ## The set of letters.
 
+  Vowels* = {'a', 'e', 'i', 'o', 'u', 'A', 'E', 'I', 'O', 'U'}
```

Comment:
This is not "standard" enough to be added IMO, for example `y` or `w` are considered vowels too in some languages and not in others.

Reply:
sadly i have to agree with @Clyybber; we should stick to standards like this: https://www.tutorialspoint.com/c_standard_library/ctype_h.htm